cedar-lean/Cedar/Thm/Data/List.lean

```diff
@@ -114,70 +114,9 @@ theorem filterMap_equiv (f : α → Option β) (xs ys : List α) :
   apply And.intro <;>
   intro b a h₃ h₄ <;>
   exists a <;>
-  simp [h₄]
-  exact h₁ h₃
-  exact h₂ h₃
-
-theorem filterMap_empty_iff_f_returns_none {f : α → Option β} {xs : List α} :
-  xs.filterMap f = [] ↔ ∀ x ∈ xs, f x = none
```

Comment:
Moved to a dedicated section for `filterMap`, and renamed `filterMap_empty_iff_all_none`.  @emina I don't think the internal development depends on this one; if I am wrong, I can undo the rename.